charts/argo-events/templates/gateway-controller-clusterrole.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
is this applying the principle of least privilege?  it looks like the role has access to everything

Reply:
@andrewgdavis obviously not :( This was just copied from the upstream.

I'd prefer to get it fixed in https://github.com/argoproj/argo-events/blob/master/hack/k8s/manifests/argo-events-cluster-roles.yaml (not sure if an issue is already filed or not), then copy it here.

Reply:
The chart has been updated. Can we close this pr?